results/proto/github.com/tektoncd/experimental/results/proto/proto/taskrun.pb.go

```diff
@@ -0,0 +1,1652 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I think this file is the same as results/proto/proto/taskrun.pb.go? We should consolidate under 1 file. I think the other one makes more sense, otherwise this Go package will resolve to "github.com/tektoncd/experimental/results/proto/github.com/tektoncd/experimental/results/proto/proto/taskrun.pb.go" which seems unnecessarily redundant.

Reply:
Yeah, will remove it